src/scss/_main.scss

```diff
@@ -188,8 +188,10 @@ nav {
     margin: 0;
     padding: 0;
 
+    color: var(--color-black);
+
     list-style-type: none;
-    background-color: var(--color-bg-dark);
+    background-color: var(--color-bg-darkest);
```

Comment:
since the color value for `--color-bg-darkest` is the same as `--color-bg-dark`, i think we can revert this

Reply:
done!